PanelSearch/main.py

```diff
@@ -20,10 +20,10 @@
 # Set up logging to include both file and console logging
```

Comment:
Is there a way we can stop logging from printing out on the console? It would be nicer for the user if logging was just in the log file rather than the console too?

https://stackoverflow.com/questions/38351934/how-to-redirect-python-logging-output-to-file-instead-of-stdout